nifi-nar-bundles/nifi-provenance-repository-bundle/nifi-persistent-provenance-repository/src/main/java/org/apache/nifi/provenance/lucene/DocsReader.java

```diff
@@ -47,9 +46,6 @@
 public class DocsReader {
     private final Logger logger = LoggerFactory.getLogger(DocsReader.class);
 
-    public DocsReader(final List<File> storageDirectories) {
-    }
-
```

Comment:
I agree that depreciation is the right thing to do in general cases like this. But given that the project is pre-1.0 and obvious intent of this class not being a public API I felt removing it would be ok. Still do but will honor the majority opinion and will deprecate.